packages/medusa/src/types/global.ts

```diff
@@ -33,14 +33,26 @@ export type Logger = _Logger & {
   warn: (msg: string) => void
 }
 
+export type sslInterface =
+{
+  ca:string|undefined
+  rejectUnauthorized:boolean|undefined
+}
+
 export type ConfigModule = {
   projectConfig: {
     redis_url?: string
 
     jwt_secret?: string
     cookie_secret?: string
+    database_host?:string|undefined
```

Comment:
issue: You should remove all the undefined you added since it is manage by `?` which makes the properties optional and therefor undefined